edge-modules/functions/samples/docker/linux/amd64/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ﻿# syntax=docker/dockerfile:1.4
 
-FROM mcr.microsoft.com/azure-functions/dotnet:3.0
+FROM mcr.microsoft.com/azure-functions/dotnet:4.0
```

Comment:
Just wondering if this should be something other than 4.0?

Is the dotnet 6 tag somehow not consistent with versioning information? Nothing you can do, I suppose. It's just quirky.

(I did try to pull mcr.microsoft.com/azure-functions/dotnet:6.0 -- and that doesn't exist)

Reply:
4.0 is dotnet 6, so we are good to go.